builds/jenkins-build-sls-app/Jenkinsfile

```diff
@@ -0,0 +1,1062 @@
+#!groovy
+
+import groovy.json.JsonOutput
+import groovy.json.JsonSlurper
+import groovy.transform.Field
+
+// To be replaced as @Field def repo_credential_id = "value" for repo_credential_id, repo_base and repo_core
+
+@Field def repo_credential_id = "jazz_repocreds"
+@Field def repo_base = "52.91.194.79"
+@Field def repo_core = "slf"
+@Field def scm_type = "gitlab"
+
+@Field def configModule
+@Field def configLoader
+@Field def scmModule
+@Field def serviceConfigdata
+@Field def events
+@Field def lambdaEvents
+@Field def serviceMetadataLoader
+@Field def utilModule
+@Field def environmentDeploymentMetadata
+@Field def sonarModule
+@Field def whiteListModule
+
+@Field def auth_token = ''
+@Field def config
+@Field def g_base_url = ''
+@Field def g_svc_admin_cred_ID = 'SVC_ADMIN'
+@Field def current_environment = ''
+@Field def isLambdaUpdateRequired = false
```

Comment:
Remove unused variables.

Reply:
Well, each stated variable is used inside code so it ain't gonna be easy to remove any of those

Reply:
Well, may be in multi-environment software they are going to be deprecated but now they are used

Reply:
Partially resolved with https://github.com/tmobile/jazz/pull/829/commits/44801449e5a2ef8d3d52478556ce9719c10d8370

Reply:
However I did not touch g_base_url, g_svc_admin_cred_ID, current_environment because they are used in some seemingly meaningful expressions all over the code